lib/config/main.go

```diff
@@ -24,6 +24,7 @@ type oplogtoredisConfiguration struct {
 	OplogV2ExtractSubfieldChanges bool          `default:"false" envconfig:"OPLOG_V2_EXTRACT_SUBFIELD_CHANGES"`
 	WriteParallelism              int           `default:"1" split_words:"true"`
 	ReadParallelism               int           `default:"1" split_words:"true"`
+	PostgresPersistenceURL        string        `envconfig:"PG_PERSISTENCE_URL"`
```

Comment:
does this default to the empty string (rest of implementation sort of implies that)?

Reply:
yeah, it does. I can make that explicit